debian/control

```diff
@@ -0,0 +1,15 @@
+Source: neofs-node
+Section: neofs
+Priority: optional
+Maintainer: NeoSPCC <tech@nspcc.ru>
+Build-Depends: debhelper-compat (= 13), git, bash-completion, dh-sequence-bash-completion
+Standards-Version: 4.5.1
+Homepage: https://github.com/nspcc-dev/neofs-node
```

Comment:
`Homepage` is https://fs.neo.org/
https://github.com/nspcc-dev/neofs-node is `Vcs-Git` or `Vcs-Browser`

Reply:
updated